tests/dummy/app/templates/application.hbs

```diff
@@ -32,6 +32,7 @@
         <div class="install-wrapper">
           <code class="install-cmd">ember install ember-cli-clipboard</code>
           {{#copy-button
+            title='copy direct'
```

Comment:
The dummy app drives the demo page. Do you mind updating this to be `title='copy to clipboard'` and adding the attr to the `{{code-block}}`?